### PR TITLE
Removed caching of Table values

### PR DESCRIPTION
**Pull request by jlstevens**
This PR allows Tables to be updated with streams by disabling the caching of cell values in TablePlot. I've tested that streams now work but I won't be confident that everything is working as it should until the tests pass.

### CODE REVIEW

**Comment by jlstevens** [comment]
One consequence of using streams is that it isn't possible for ``TablePlot`` to adjust the size of the table based on future (unknown) values. This will always be an issue using ``DynamicMap`` so I only want to point out this problem as I'm not sure how it can be solved.

File: holoviews/plotting/mpl/tabular.py [hunk 1]
```diff
@@ -117,7 +115,7 @@ def initialize_plot(self, ranges=None):
             for col in range(element.cols):
                 if summarize and row > half_rows:
                     adjusted_row = (element.rows - self.max_rows + row)
-                cell_value = self.cell_values[self.keys[-1]][(row, col)]
+                cell_value = element.pprint_cell(row, col)
```
Comment (philippjfr) [inline holoviews/plotting/mpl/tabular.py]: Shouldn't both of these calls also pass the return value through the ``pprint_value`` method on TablePlot, I'm not sure why both the element and the plot need a pprint method but I assume there was a reason.

Comment (jlstevens) [inline holoviews/plotting/mpl/tabular.py]: Well spotted! I've just pushed a commit to fix this.

**Comment by philippjfr** [comment]
Looks good now. Happy to merge when tests pass.

**Pull request merged**